CASMTRIAGE-8092 (#5975)
Cray-HPE/docs-csm: operations/iuf/workflows/configuration.md

    - Generate SAT S3 credentials
    - Configure system revision information via `sat setrev`

> **`NOTE`** If the system is using Customer High Speed Network(CHN), uncomment the following playbook in `management-bootprep.yaml`.

Reviewer: ```suggestion
**`NOTE`** If the system is using the Customer High Speed Network (CHN), uncomment the following playbook in the `management-bootprep.yaml` file.
```

>  ```bash

Reviewer 2: ```suggestion
 ```yaml
```

#BEGIN_CHN_SUPPORT
  - name: enabled_chn-{{csm.version}}
    playbook: enable_chn.yml
    product:
    name: csm
    version: "{{csm.version}}"
#END_CHN_SUPPORT         
```

> In order to check whether the system is using Customer High Speed Network(CHN), refer to the steps mentioned [here](../../../operations/network/management_network/chn_test.md#procedure-to-check-chn)

Reviewer: ```suggestion
In order to check whether the system is using the Customer High Speed Network (CHN), see the [procedure to check CHN](../../../operations/network/management_network/chn_test.md#procedure-to-check-chn)
```
